release/ray_release/test_automation/state_machine.py

```diff
@@ -272,3 +274,51 @@ def comment_blamed_commit_on_github_issue(self) -> None:
             f"found by bisect job https://buildkite.com/{BUILDKITE_ORGANIZATION}/"
             f"{BUILDKITE_BISECT_PIPELINE}/builds/{bisect_build_number}"
         )
+
+    def _trigger_bisect(self) -> None:
```

Comment:
should there be some unit tests that should be moved too, like to `test_state_machine.py`?

Reply:
there is only one test file for all classes of the state machine (`test_state_machine.py`; the unit test for bisect already exists here https://github.com/ray-project/ray/blob/master/release/ray_release/tests/test_state_machine.py#L249